server/routes/creatingLicences/handlers/initialMeetingTime.ts

```diff
@@ -13,13 +14,19 @@ export default class InitialMeetingTimeRoutes {
 
   POST = async (req: Request, res: Response): Promise<void> => {
     const { licenceId } = req.params
+    const { licence } = res.locals
     const { username } = res.locals.user
+
     await this.licenceService.updateAppointmentTime(licenceId, req.body, username)
 
     if (req.query?.fromReview) {
-      res.redirect(`/licence/create/id/${licenceId}/check-your-answers`)
-    } else {
-      res.redirect(`/licence/create/id/${licenceId}/additional-conditions-question`)
+      return res.redirect(`/licence/create/id/${licenceId}/check-your-answers`)
+    }
```

Comment:
Is this right? Always goes to check-your-answers when fromReview === true? Not sure why it used to go to additonal-conditions?

Reply:
Yea thats right, I dont think the logic around fromReview has changed here? Go to check your answers if fromReview = true, otherwise decide whether to go to additional conditions or pss conditions based on licence type